addons/web/static/src/core/utils/urls.js

```diff
@@ -58,6 +59,30 @@ export function url(route, queryParams, options = {}) {
     return `${prefix}${route}${queryString}`;
 }
 
+export function urlImage(model, id, field, { filename, width, height, unique } = {}) {
+    let route = `/web/image/${model}/${id}/${field}`;
+    if (width && height) {
+        route = route + `/${width}x${height}`;
+    }
+    if (filename) {
+        route = route + `/${filename}`;
+    }
+    const urlParams = {};
+    if (unique) {
+        if (unique instanceof DateTime) {
+            urlParams.unique = unique.ts;
+        } else {
+            const DateTimeFromUnique = DateTime.fromSQL(unique);
+            if (DateTimeFromUnique.isValid) {
+                urlParams.unique = DateTimeFromUnique.ts;
+            } else {
+                urlParams.unique = unique;
+            }
+        }
```

Comment:
Maybe improve (or remove) `imageCacheKey` (all its usage should be replaced by this new function instead)?